test/cmdlineTests.sh

```diff
@@ -312,6 +325,13 @@ printTask "Running general commandline tests..."
 
         # Strip trailing slash from $tdir.
         tdir=$(basename "${tdir}")
+        if [[ $no_smt == true ]]
+        then
+            if [[ "$tdir" =~ .*model_checker.* ]]; then
```

Comment:
I'd include the underscore in the condition to make sure we only match full words.
```suggestion
            if [[ $tdir =~ .*model_checker_.* ]]; then
```

Also, I think quotes are not needed on the left side inside `[[ ]]`.

Reply:
done